### PR TITLE
Template

### PR DESCRIPTION
**Pull request by Liliya-V**

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,100 @@
+*{
+    box-sizing:border-box;
+}
+
+body {
+    background-color :#f2f2f2;
+    font-size: 62.5%;
+    font-family: HKGrotesk, sans-serif;
+    height: 100vh;
```
Comment (lucyholubova) [inline stylesheet.css:9]: do not set hight on body

File: homepage.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en"><head>
```
Comment (lucyholubova) [inline homepage.html]: i would put <head> under html

File: homepage.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <meta http-equiv="content-type" content="text/html; charset=windows-1252">
```
Comment (lucyholubova) [inline homepage.html]: this could be replace with 
<meta charset="UTF-8">

File: homepage.html (hunk 2 above)
Comment (Liliya-V) [reply homepage.html]: done

File: stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    height: 50vh;
+    width: 100vw;
+    position:relative;
+    border-top: 5px solid #f2f2f2;
+    border-left: 5px solid #f2f2f2;
+    border-right: 5px solid #f2f2f2;
+
+}
+nav {
+    width: 98%;
```
Comment (lucyholubova) [inline stylesheet.css:24]: spoke about the maybe 100% width

File: stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+    width: 100%;
+    height: 90%;
+    /*display: flex;*/
+    /*justify-content: center;*/
+    /*align-content: center;*/
+    position: absolute;
+    top: 5%;
+    z-index: 2;
+}
+.arrows {
```
Comment (lucyholubova) [inline stylesheet.css:57]: i think if you get rif of the height the arrows will change the position completely

File: stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+    color: #404040;
+    font-family: "NoeText", sans-serif;
+    font-size: 3vw;
+    z-index: 3;
+    position: absolute;
+    bottom: -60px;
+    left: 25%;
+    margin: 0;
+}
+.social {
```
Comment (lucyholubova) [inline stylesheet.css:86]: also once i got rid of height these dissapeard not sure why

File: stylesheet.css (hunk 1 above)
Comment (Liliya-V) [reply stylesheet.css:9]: Set height relative to a viewport so when viewport height is changed the page remains not scrollable.

File: stylesheet.css (hunk 4 above)
Comment (Liliya-V) [reply stylesheet.css:24]: Rethought it, parent (Hometop) width is set to 100vw which is 100% basically . Left nav 98% as it doesn't affect the page width.

**Review by sjr-mayden — APPROVED** [review]

File: homepage.html [hunk 7]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+     <meta charset="UTF-8">
+     <title>Portfolio Liliya Voevodina.html</title>
+     <link href="normalizeportfolio.css" rel="stylesheet" type="text/css">
```
Comment (neilbulloch) [inline homepage.html:6]: If you placed the normalize.css into a CSS folder you can just point to the file inside a folder like this : folder_name / file_name  ex: cssportfolio/normalize.css

File: homepage.html [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+     <link href="stylesheet.css" rel="stylesheet" type="text/css">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   </head>
+      <body>
+       <div class="hometop">
```
Comment (neilbulloch) [inline homepage.html:12]: consider using 1 tab for indentation just to make the structure a little clearer for you and others who read the code :)

File: homepage.html [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+        <nav>
+          <img class="mypicture" src="IMG_9169.jpg" alt="Liliya">
+          <a href="#">Contacts</a>
+          <a href="#">Portfolio</a>
+          <a href="#">About me</a>
+          <a href="#">Home</a>
+        </nav>
+        <h1>
+
```
Comment (neilbulloch) [inline homepage.html:21]: Your h1 header is empty at the moment.

File: stylesheet.css [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+    box-sizing:border-box;
+}
+
+body {
+    background-color :#f2f2f2;
+    font-size: 62.5%;
+    font-family: HKGrotesk, sans-serif;
+    height: 100vh;
+    width: 100vw;
+    position: relative;
```
Comment (neilbulloch) [inline stylesheet.css:11]: Is there a reason you set the position of the body relative?

**Pull request merged**